tools/mtouch/Application.cs

```diff
@@ -2175,7 +2175,14 @@ public void BundleAssemblies ()
 					return false;
 				if (PackageManagedDebugSymbols)
 					return false;
-				if (IsInterpreted (Assembly.GetIdentity (path)))
+				/* FIXME: should be `if (IsInterpreted (Assembly.GetIdentity (path)))`.
+				 * The problem is that in mixed mode we can't do the transition
+				 * between "interp"->"aot'd methods using gsharedvt", so we
+				 * fall back to the interp and thus need the IL not to be
+				 * stripped out. Once Mono supports this, we can add back the
+				 * more precise check.
+				 * See https://github.com/mono/mono/issues/11942 */
+				if (UseInterpreter)
```

Comment:
In theory, the methods which have aot code can still be stripped, i.e. the non-generic methods.

Reply:
afaik `mono-cil-strip` doesn't support this